UPEI-Android/group-project-mindjogg
# Implemented Drawer navigation

## Source 
Closes #115 

### Summary of PR 

The PR introduce drawer navigation for the MindJoGG 
 
### Review and Testing Steps

1.Pull this branch
2. Run  ``` cd app/client/mindjogg ```
3. create a .env file and get the values from the shared doc
4. run ``` npm install```
5. run ```npm start```
6. You will see the drawer nav
7. check if all screens are accessible or not 
  

### Additional Information (If applicable)
only the main screen of each mini app will have hamburger menu


### Notes

The account management main screen will not have hamburger menu
